### PR TITLE
`treehouses picture` (fixes #1491)

### LINKED ISSUE #1491: treehouses terminal image viewer
Terminal Image Viewer has been added to treehouses release-136 under the command `tiv`, we should add a cli command to work with it.

https://github.com/stefanhaustein/TerminalImageViewer

### PR DESCRIPTION
**Pull request by ComputerOnFire**
adds `treehouses picture`, a command that utilizes `tiv` (Terminal Image Viewer) to display a picture over the terminal.

- fixes #1491

### CODE REVIEW

**Comment by kjong** [comment]
What release of treehouses should we be using to test this?

I'm on 135 and it doesn't seem to find tiv.

**Comment by darthnoward** [comment]
tried a few times on different images, both url and locally, both png and jpg.
also tried two different terminals (konsole and urxvt)
got error messages similar to this
```
root@treehouses:~# cli/cli.sh picture https://treehouses.io/images/logo.png

[CImg] *** CImgIOException *** [instance(0,0,0,0,(nil),non-shared)] CImg<unsigned char>::load(): Failed to recognize format of file '/tmp/I3ZtwI11.png'.
File format is not recognized for 'https://treehouses.io/images/logo.png'
```

**Comment by ComputerOnFire** [comment]
seems to be missing the `imagemagick` dependency. i'm currently fixing this in the builder

**Comment by ComputerOnFire** [comment]
turns out we need imagemagick on the pi, not during compilation. im implementing it in the builder but you can test it locally with
`apt-get install imagemagick`

**Comment by ComputerOnFire** [comment]
works on a fresh image with built with imagemagick

File: modules/picture.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    case "$1" in
+      "")
+        picture_help
+      ;;
+    
+      *)
+        tiv "$1"
+      ;;
+    esac
+}
```
Comment (dogi) [inline modules/picture.sh:11]: check for tiv and imagemagick

**Pull request merged**